package.json

```diff
@@ -32,9 +32,17 @@
     "url": "git://github.com/ramda/ramda.git"
   },
   "main": "dist/ramda.js",
+  "bin": {
+    "jscs": "node_modules/jscs/bin/jscs",
+    "jshint": "node_modules/jshint/bin/jshint",
+    "mocha": "node_modules/mocha/bin/mocha"
+  },
```

Comment:
This is an abuse, I believe. It will add symlinks for these to a users __node_modules/.bin__ directory when she runs `npm install ramda`. Let's just specify the full paths.

By the way, you can use __node_modules/.bin/jscs__ rather than __node_modules/jscs/bin/jscs__ because JSCS specifies [`"bin"`][1] in its [__package.json__][2]. The same applies for the other two.


[1]: https://docs.npmjs.com/files/package.json#bin
[2]: https://github.com/jscs-dev/node-jscs/blob/v1.13.1/package.json#L99-L101